chart/values.yaml

```diff
@@ -0,0 +1,29 @@
+
+image:
+  repository: estafette/estafette-cloudflare-dns
+  pullPolicy: IfNotPresent
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+rbac:
+  # If true, create & use RBAC resources
+  create: false
```

Comment:
You may want to add this field in case rbac is false:
```
  # Defines the serviceAccountName to use when `rbac.create=false`
  serviceAccountName: default
```

Then in the deployment:

```
serviceAccountName: {{ if .Values.rbac.create }}{{ template "fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
```

Reply:
I saw that elsewhere, but imho it doesn't really matter whether you inject a custom serviceaccount without permissions or the default serviceaccount. Or does the default service account have some permissions you would otherwise miss?